notebooks/gradient_decent.ipynb

```diff
@@ -11,46 +11,32 @@
    "cell_type": "markdown",
```

Comment:
<p>The font size is too large</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/m2lines/L96_demo/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>